public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -193,16 +201,58 @@ export class DashboardModel implements TimeModel {
     );
   }
 
+  get title(): string {
+    return store.getState().dashboard.title;
+  }
+
+  set title(newTitle: string) {
+    dispatch(updateDashboard({ title: newTitle }));
+  }
+
+  get liveNow(): boolean {
+    return store.getState().dashboard.liveNow;
+  }
+
+  set liveNow(live: boolean) {
+    dispatch(updateDashboard({ liveNow: live }));
+  }
+
+  get graphTooltip(): DashboardCursorSync {
+    return store.getState().dashboard.graphTooltip;
+  }
+
+  set graphTooltip(newVal: DashboardCursorSync) {
+    dispatch(updateDashboard({ graphTooltip: newVal }));
+  }
+
+  get description(): string {
+    return store.getState().dashboard.description;
+  }
+
+  set description(newDescription: string) {
+    dispatch(updateDashboard({ description: newDescription }));
+  }
```

Comment:
A bit worried about this migration approach, Hugo tried the same thing for the old variables system and it turned into a complexity nightmare and complicated so many tests. it was later abandoned. DashboardModel is used in a million tests and is so nice to work with as it's so standalone and decoupled. You can use multiple models at the same time etc.

I think the approach outlined here https://github.com/grafana/grafana/pull/41923 is better where the redux updates the dashboard model instead (so both live side by side but you only update via redux).

It feels at least less surprising that redux updates the legacy model that is in redux already (via getModel). rather than every class instance reads and updates from a global redux singleton.